src/SamplesApp/SamplesApp.netcoremobile/Android/AssemblyInfo.Android.cs

```diff
@@ -13,3 +13,4 @@
 [assembly: UsesPermission("android.permission.RECORD_AUDIO")]
 [assembly: UsesPermission("android.permission.CAMERA")]
 [assembly: UsesPermission("android.permission.WRITE_EXTERNAL_STORAGE")]
+[assembly: UsesPermission("android.hardware.sensor.proximity")]
```

Comment:
Not important, but there is another AssemblyInfo file for the Xamarin-based project.

Reply:
Actually turns out it is not needed, so I removed it here